includes/class-sensei-course.php

```diff
@@ -224,6 +224,14 @@ public function add_showcase_courses_upsell() {
 	 * @internal
 	 */
 	public function showcase_courses_screen() {
+		// Get the price of Pro. Return if it's not available.
+		$sensei_pro_product = Sensei_Extensions::instance()->get_extension( 'sensei-pro' );
+		if ( ! $sensei_pro_product ) {
+			return;
```

Comment:
Maybe instead of returning, we could write something like `"-"` as an error, so the user could see the rest of the content.

Reply:
Nice idea! I've changed it in 98e672243a7bfc9286af0f76fd704457cb317bbd